src/slider/slider.js

```diff
@@ -30,6 +30,7 @@
     this.element_ = element;
     // Browser feature detection.
     this.isIE_ = window.navigator.msPointerEnabled;
+    this.isEdge_ = !this.isIE_ && !!window.StyleMedia;
```

Comment:
A compromise would be to fix the stylesheet only, this way we at least get the Thumb back.